Install script for debian based distributions added.

kiliansinger opened the pull request.

kiliansinger committed:
Install script for debian based distributions added.

aharjula commented:
Good start, but I would rather see this file in the docs folder and with very specific warnings that this is for Debians with such an such tomcat/openjdk version. It would also be nice if the script would not try to clone the repository on top of an existing one it should just pull in that case. Also overwriting the pool.conf if it already exists should be confirmed.

It's nice that someone is interested in MaximaPools improvement, now if we could find someone to write a script that asks how much of the available memory should be given to the pool and would then measure the memory usage of a single maxima (with some relatively big inputs) and would tune the pool size parameters accordingly. The script should also execute simultaneous startups of multiple maximas to find out when things slow down so that the start-limit could be given a reasonable value, the current default 5 is quite low for most of the real server installs but can't be given any higher values due to the slowdown caused on lower end machines especially VMs.

kiliansinger committed:
Added check for existing git and then only doing git pull.
Also added cp -i (interactive mode) to let user choose
if he wants to overwrite config files.

kiliansinger committed:
moved install script

kiliansinger committed:
Added notice and question.

kiliansinger commented:
Incoorporated notes.

aharjula commented:
Thank you.
